tests/helics/application_api/FederateTests.cpp

```diff
@@ -46,6 +47,58 @@ TEST(federate_tests, federate_initialize_tests)
     Fed = nullptr;  // force the destructor
 }
 
+TEST(federate_tests, federate_initialize_tests_env)
+{
+    setEnvironmentalVariable("HELICS_LOG_LEVEL", "connections");
+    helics::FederateInfo fi(CORE_TYPE_TO_TEST);
+    fi.coreInitString = "--autobroker";
+
+    auto Fed = std::make_shared<helics::Federate>("test1", fi);
+
+    auto core = Fed->getCorePointer();
+    ASSERT_TRUE((core));
+
+    auto cloglevel = core->getIntegerProperty(helics::local_core_id, helics_property_int_log_level);
+    EXPECT_EQ(cloglevel, helics_log_level_connections);
+
+    Fed->enterExecutingMode();
+    EXPECT_EQ(Fed->getIntegerProperty(helics_property_int_log_level), helics_log_level_connections);
+    EXPECT_TRUE(Fed->getCurrentMode() == helics::Federate::modes::executing);
+
+    Fed->finalize();
+    clearEnvironmentalVariable("HELICS_LOG_LEVEL");
+    clearEnvironmentalVariable("HELICS_BROKER_LOG_LEVEL");
```

Comment:
```suggestion
    clearEnvironmentVariable("HELICS_LOG_LEVEL");
```

It doesn't look like `HELICS_BROKER_LOG_LEVEL` is set in this test.

Reply:
probably a relic from the test I copied from